Code/GraphMol/Depictor/templates.smi

```diff
@@ -0,0 +1,78 @@
+c1cc2ccc1-c1ccc(cc1)-c1ccc(cc1)-c1ccc(cc1)-c1ccc(cc1)-c1ccc-2cc1 |(-1.23952,-2.13951,;-0.00687016,-1.42979,;-0.00517751,-0.007368,;-1.23623,0.705237,;-2.46888,-0.00448054,;-2.47057,-1.4269,;-3.45469,-3.12841,;-4.68654,-3.83952,;-4.68654,-5.26195,;-3.45479,-5.97316,;-2.22294,-5.26204,;-2.22285,-3.83962,;-2.47077,-7.67477,;-2.46927,-9.09719,;-1.23673,-9.8071,;-0.00567535,-9.0946,;-0.00716886,-7.67228,;-1.23972,-6.96236,;1.96009,-9.0948,;1.96188,-7.67248,;3.19452,-6.96276,;4.42548,-7.67536,;4.42379,-9.09778,;3.19114,-9.8075,;5.4096,-5.97385,;6.64145,-5.26274,;6.64145,-3.84032,;5.4096,-3.12911,;4.17775,-3.84032,;4.17775,-5.26274,;4.42568,-1.4275,;3.19452,-2.13991,;1.96198,-1.42999,;1.96058,-0.00756713,;3.19164,0.704839,;4.42418,-0.00507794,)|
```

Comment:
is the aromaticity important to these templates? I don't think that coordgen uses bond orders when matching templates, so we can probably drop that here. I also don't know if atomic number even matters!